src/target.zig

```diff
@@ -431,6 +431,13 @@ pub fn defaultFunctionAlignment(target: std.Target) Alignment {
     };
 }
 
+pub fn minFunctionAlignment(target: std.Target) Alignment {
+    return switch (target.cpu.arch) {
+        .riscv64 => .@"2",
```

Comment:
This should be true for all RISCs I believe. Would you mind adding that in then?

Reply:
No problem.

Reply:
Done!